runtime/jnichk/jnicheck.c

```diff
@@ -477,7 +477,10 @@ jniCheckCallV(const char* function, JNIEnv* env, jobject receiver, UDATA methodT
 			if (argNum !=3) j9tty_printf(PORTLIB, ", ");
 		}
 		switch (*sigArgs) {
-		case 'L':
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+		case 'Q': /* fall through */
+#endif /* #if defined(J9VM_OPT_VALHALLA_VALUE_TYPES) */
+		case 'L': /* fall through */
```

Comment:
I don't think the comment `/* fall through */` is necessary (or helpful) in cases like this where there's no code between case labels.

Reply:
removed in all cases in 9987875